src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1587,6 +1734,423 @@ void addStreamHDFSCommand(std::string masterIP, int connFd, std::string &hdfsSer
+        return false;
+    }
+    resultWr = write(connFd, Conts::CARRIAGE_RETURN_NEW_LINE.c_str(), Conts::CARRIAGE_RETURN_NEW_LINE.size());
+    if (resultWr < 0) {
+        frontend_logger.error("Error writing to socket");
+        *loop_exit_p = true;
+        return false;
+    }
+
+    char chunkSize[FRONTEND_DATA_LENGTH + 1];
+    memset(chunkSize, 0, FRONTEND_DATA_LENGTH + 1);
+    read(connFd, chunkSize, FRONTEND_DATA_LENGTH);
+    std::string chunkSizeS(chunkSize);
+    chunkSizeS = Utils::trim_copy(chunkSizeS);
+    frontend_logger.info("received chunk Size engine: " + chunkSizeS);
+
+    int newGraphID;
+    bool graphExits = false;
+    std::string checkQuery = "SELECT idgraph FROM graph WHERE upload_path = \"" + path + "\";";
+    auto result = sqlite->runSelect(checkQuery);
+
+    if (!result.empty() && !result[0].empty()) {
+        int existingId = std::stoi(result[0][0].second);
+        frontend_logger.info("Graph already exists with ID: " + std::to_string(existingId));
+
+        std::string resume_msg = "There exists a graph with the file path, would you like to resume?";
+        resultWr = write(connFd, resume_msg.c_str(), resume_msg.length());
+        if (resultWr < 0) {
+            frontend_logger.error("Error writing to socket");
+            *loop_exit_p = true;
+            return false;
+        }
+        resultWr = write(connFd, Conts::CARRIAGE_RETURN_NEW_LINE.c_str(), Conts::CARRIAGE_RETURN_NEW_LINE.size());
+        if (resultWr < 0) {
+            frontend_logger.error("Error writing to socket");
+            *loop_exit_p = true;
+            return false;
+        }
+
+        char resume[FRONTEND_DATA_LENGTH + 1];
+        memset(resume, 0, FRONTEND_DATA_LENGTH + 1);
+        read(connFd, resume, FRONTEND_DATA_LENGTH);
+        std::string resumeS(resume);
+        resumeS = Utils::trim_copy(resumeS);
+
+        if (resumeS == "y") {
+            resume_msg = "Graph Id to resume?";
+            resultWr = write(connFd, resume_msg.c_str(), resume_msg.length());
+            if (resultWr < 0) {
+                frontend_logger.error("Error writing to socket");
+                *loop_exit_p = true;
+                return false;
+            }
+            resultWr = write(connFd, Conts::CARRIAGE_RETURN_NEW_LINE.c_str(), Conts::CARRIAGE_RETURN_NEW_LINE.size());
+            if (resultWr < 0) {
+                frontend_logger.error("Error writing to socket");
+                *loop_exit_p = true;
+                return false;
+            }
+
+            char resumeGraphId[FRONTEND_DATA_LENGTH + 1];
+            memset(resumeGraphId, 0, FRONTEND_DATA_LENGTH + 1);
+            read(connFd, resumeGraphId, FRONTEND_DATA_LENGTH);
+            std::string resumeGraphIdS(resumeGraphId);
+            resumeGraphIdS = Utils::trim_copy(resumeGraphIdS);
+
+            newGraphID = stoi(resumeGraphIdS);
+            graphExits = true;
+            frontend_logger.info("1947 GrpahID: " + to_string(newGraphID));
```

Comment:
Please create a meaningful log message here. What do you mean by `1947 GraphID: ` ?